recipes/libbacktrace/all/conanfile.py

```diff
@@ -79,7 +85,8 @@ def _configure_autotools(self):
             return self._autotools
         self._autotools = AutoToolsBuildEnvironment(self, win_bash=tools.os_info.is_windows)
         self._autotools.libs = []
-        if self.settings.compiler == "Visual Studio" and tools.Version(self.settings.compiler.version) >= "12":
+        if (self.settings.compiler == "Visual Studio" and tools.Version(self.settings.compiler.version) >= "12") or \
+           str(self.settings.compiler) == "msvc":
```

Comment:
why is there no version check for msvc ?

Reply:
I don't trust stability of msvc versionning in settings.yml currently. Moreover, it begins at Visual Studio 2015.